### ISSUE TITLE
Reproducing results on LEVIR and logs

### ISSUE THREAD

**Issue by robmarkcole** [AndreaCodegoni/Tiny_model_4_CD#18]
Using the supplied training scripts and dataset, I here share the logs of the training run on LEVIR. 100 epochs appears to be well chosen as learning plateaus here. 

<img width="1477" alt="image" src="https://github.com/AndreaCodegoni/Tiny_model_4_CD/assets/11855322/442b0e2e-c35f-43a4-97fb-ad484ecafc2b">

I trained on a V100 Lightning.ai instance in around 4 hours:

```
Validation phase summary
Loss for epoch 99 is 0.0209393590421314
IoU class change for epoch 99 is 0.8350830680701221
F1 class change for epoch 99 is 0.9101310212289551
```

**Comment by robmarkcole** [comment]
I've now reproduced the training in the lightning framework to enable the comparison in https://github.com/AndreaCodegoni/Tiny_model_4_CD/issues/17

Having resolved differences in dataset and preprocessing in my codebase I achieve at 100 epochs:
```
'test_iou': 0.8267
'test_f1': 0.9051
```

<img width="1807" alt="image" src="https://github.com/AndreaCodegoni/Tiny_model_4_CD/assets/11855322/8727bef7-0adc-42fe-bc30-c43dc651441a">

The predictions do not look quite as clean as the paper however, but perhaps they look reasonable?

![output](https://github.com/AndreaCodegoni/Tiny_model_4_CD/assets/11855322/0a549cf1-9585-41c6-a6f9-23c554a9d277)

![output2](https://github.com/AndreaCodegoni/Tiny_model_4_CD/assets/11855322/ff1d96a2-fd17-493d-8b15-d9e5ae0bb21f)

**Comment by robmarkcole** [comment]
I reran without the bandwise normalisation and the results are more or less the same:
```
'test_iou': 0.821
'test_f1': 0.902
```
